packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -314,33 +312,52 @@ function buildTroubleshooterData({
   return troubleshooterData;
 }
 
+export type PlatformParams = {
+  platformClientId?: string;
+  platformCancelUrl?: string;
+  platformBookingUrl?: string;
+  platformRescheduleUrl?: string;
+  platformBookingLocation?: string;
+};
+
+export type BookingHandlerInput = {
+  bookingData: Record<string, any>;
+  userId?: number;
+  // These used to come from headers but now we're passing them as params
+  hostname?: string;
```

Comment:
Maybe we can group `hostname` and `forcedSlug` under `params` property in `BookingHandlerInput` and also have `noEmail` there? Right now `noEmail` is in the `bookingData` and also group properties like `_isDryRun` there - properties that are not used to create booking per se but impact the process of it. Right now "util" data like `noEmail` is in `bookingData` but `forcedSlug` on the root level. Maybe requires too much effort rn but just noting this.